save-demo/src/main/kotlin/com/saveourtool/save/demo/utils/KubernetesUtils.kt

```diff
@@ -128,17 +144,26 @@ fun jobNameForDemo(demo: Demo) = with(demo) { "demo-${organizationName.lowercase
 private fun demoAgentContainerSpec(
     imageName: String,
     agentDownloadUrl: String,
+    demo: Demo,
     kubernetesSettings: KubernetesConfig,
+    agentConfig: ConfigProperties.AgentConfig,
 ) = Container().apply {
     name = "save-demo-agent-pod"
     image = imageName
     imagePullPolicy = "IfNotPresent"
 
     val envOptions = sequenceOf(
```

Comment:
does these variables are really `env` variables? and could be accessed via system calls? or it's just a such name?

Reply:
Added `export`

Reply:
please take/duplicate the implementation from orchestrator